### DISCUSSION TITLE
Trigger a budibase button using an embed.

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7654] [Help] [answered]
Hello everyone.
How to trigger a budibase button using an embed.
For example i want to click on save button but in an indirect way

**Comment by melohagan** [comment]
Are you trying to trigger some action on an event? What is the event?

**Reply by rafikApp** [reply]
Hello @melohagan 
Yes i want to trigger save row action using onclick action ( embed )

**Comment by melohagan** [comment, marked as answer]
The clickable containers has been released recently, so you can put an invisible clickable container around an embed if that helps?

Otherwise, custom components is going to be released in the next week or two, which will probably be a better option. I noticed you rely heavily on Embed components, so I think the custom components will be very useful for yourself!

**Reply by rafikApp** [reply]
Ok @rafikApp thank you 
and the custom components can solve the signature problem ?

**Reply by melohagan** [reply]
Yep! Using code from this repo here: https://github.com/SharifClick/svelte-canvas-draw

It's not released yet, but here's a sneak peak of what could be possible with custom components:
![switch](https://user-images.githubusercontent.com/101575380/190162872-e515db5f-d281-4e44-9fe8-f665b9330e5e.gif)

You would also need to hook this up to bindings to do something with the image, but more to come in the near future!

**Reply by rafikApp** [reply]
Hi @melohagan.
How can we add the custom components ?

**Reply by melohagan** [reply]
Hey @rafikApp You can see a list of custom components here: https://github.com/Budibase/plugins

And you can import them via these instructions: https://docs.budibase.com/docs/custom-plugin#importing-plugins

**Reply by rafikApp** [reply]
Thank you @melohagan 
And what about signature ?

**Reply by melohagan** [reply]
@shogunpurple Had been working on a custom signature component! I don't think it's quite finished yet.

**Reply by shogunpurple** [reply]
Yep, not finished yet. Will get round to finishing it off fairly soon

**Reply by rafikApp** [reply]
Hi @melohagan @shogunpurple 
Any news about the signature component.
Thank you for your efforts
